bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx70-en.html

```diff
@@ -46,21 +52,23 @@ <h2 class="c-emphasis-box-title">
         <div class="show-fxa-supported-signed-out show-fxa-default show-fxa-not-fx">
           {{ monitor_button(
             entrypoint=_entrypoint,
+            entrypoint_experiment=_utm_campaign,
```

Comment:
Hmm, should the default WNP be classed as control, or should you create a different template to serve as control?